packages/e2e-tests/scripts/buildkite_run_fe_tests.js

```diff
@@ -40,47 +40,53 @@ function run_fe_tests(CHROME_BINARY_PATH) {
 
   process.env.RECORD_REPLAY_DISPATCH_SERVER = "wss://dispatch.replay.io";
   process.env.REPLAY_BROWSER_BINARY_PATH = CHROME_BINARY_PATH;
+  process.env.REPLAY_CHROMIUM_EXECUTABLE_PATH = CHROME_BINARY_PATH;
+  process.env.RECORD_REPLAY_PATH = CHROME_BINARY_PATH;
+  // process.env.RECORD_REPLAY_DIRECTORY = 
   process.env.AUTHENTICATED_TESTS_WORKSPACE_API_KEY = process.env.RECORD_REPLAY_API_KEY;
   process.env.PLAYWRIGHT_TEST_BASE_URL = "https://app.replay.io";
 
   // Generate new recordings for known-passing tests with the new chromium build.
   const htmlFiles = [
-    "authenticated_comments.html",
-    "authenticated_logpoints.html",
     "doc_minified_chromium.html",
-    "doc_recursion.html",
-    "doc_rr_console.html",
-    "doc_rr_preview.html",
-    "doc_rr_region_loading.html",
-    "doc_stacking_chromium.html",
-    "rdt-react-versions/dist/index.html",
+    "doc_prod_bundle.html",
+    // "authenticated_comments.html",
```

Comment:
I assume you don't mean to submit this?

Reply:
Nope - for testing only!
Also I'm reverting this whole thing since I was actually chasing the wrong thing in this first patch.